packer/resources/features/logstash/install.sh

```diff
@@ -18,9 +18,22 @@ ln -s /opt/features/logstash/logstash /opt/logstash
 # TODO could use the handy substitution script once #23 is merged
 ESCAPED_USER=$(echo $USER | sed -e 's/[\/&]/\\&/g')
 ESCAPED_CONFIGFILE=$(echo $CONFIGFILE | sed -e 's/[\/&]/\\&/g')
+
+# Upstart
+if [ -d "/etc/init" ]; then
```

Comment:
There is still an `/etc/init` directory on vivid as far as I can see. Does it make sense to do both?